android-bindings/lib-wrapper/.gitignore

```
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
```

Optional/Equivalent: Some of these are in the top-level `.gitignore`, so we can dedupe them.
```suggestion
/build
/captures
.gradle
.externalNativeBuild
.cxx
local.properties
```

---

Also, did you mean  `*.cxx`?

---

The `lib-wrapper` is a standalone android project with a standard auto-included `.gitignore`. I wouldn't want to touch it because the change does not really affect anything right now, but if we want to move the project we will have to revert the above-proposed changes.